src/libs/sfdk/buildengine.cpp

```diff
@@ -914,6 +915,9 @@ BuildEngineManager::BuildEngineManager(QObject *parent)
     Q_ASSERT(!s_instance);
     s_instance = this;
 
+    QHostInfo::lookupHost(QHostInfo::localHostName(), this,
+            &BuildEngineManager::completeHostNameLookup);
```

Comment:
QHostInfo::localHostName already returns the local hostname, and we are not interested in the IP address, so what's the point of the lookup?

Reply:
localHostName returns just the unqualified host name, lookup needs to be done to get the possible FQDN.

Reply:
I'm not sure getting the fully qualified name is worth the extra hassle, but ok... :)